### PR TITLE
Fixes Issue #482 (issue with multidimensional sampling)

### PR DESCRIPTION
**Pull request by amrods**
There was a repeated argument for the rng in line 393. Easy fix for my first PR :)

### CODE REVIEW

**Comment by nalimilan** [comment]
Thanks. Could you add a test too?

File: test/sampling.jl [hunk 1]
```diff
@@ -170,6 +170,13 @@ check_sample_norep(a, (3, 12), 0; ordered=false)
 a = sample(3:12, 5; replace=false, ordered=true)
 check_sample_norep(a, (3, 12), 0; ordered=true)
 
+# tests of multidimensional sampling
+
+a = sample(1:4, (2, 2); replace=false)
+@test sort(vec(a)) == collect(1:4)
```
Comment (nalimilan) [inline test/sampling.jl]: Could you use `check_sample_norep` here as above?

Comment (amrods) [reply test/sampling.jl]: what is `check_sample_norep` supposed to do?

Comment (nalimilan) [reply test/sampling.jl]: AFAIK it just does a few consistency checks. That allows testing general cases even if the result is random (by definition).

Comment (amrods) [reply test/sampling.jl]: Maybe I'm not understanding how I have to use it, I'm getting this when I run it locally:
````
julia> a = sample(1:4, (2, 2); replace=false)
2×2 Array{Int64,2}:
 4  2
 3  1

julia> check_sample_norep(a, (1, 4), 5.0e-3; ordered=false)
Test Failed at test/sampling.jl:118
  Expression: isapprox(proportions(bj, vmin:vmax), p0, atol=ptol)
   Evaluated: isapprox([0.0, 0.5, 0.0, 0.5], [0.25, 0.25, 0.25, 0.25]; atol=0.005)
ERROR: There was an error during testing
````

Comment (nalimilan) [reply test/sampling.jl]: Replace `5.0e-3` with `0`?

Comment (amrods) [reply test/sampling.jl]: alright, done

File: test/sampling.jl [hunk 2]
```diff
@@ -170,6 +170,13 @@ check_sample_norep(a, (3, 12), 0; ordered=false)
 a = sample(3:12, 5; replace=false, ordered=true)
 check_sample_norep(a, (3, 12), 0; ordered=true)
 
+# tests of multidimensional sampling
+
+a = sample(1:4, (2, 2); replace=false)
```
Comment (nalimilan) [inline test/sampling.jl]: Now you can use a longer range.

Comment (amrods) [reply test/sampling.jl]: I think the other test is redundant, should I remove it?

Comment (nalimilan) [reply test/sampling.jl]: As you wish, but it doesn't really hurt.

Comment (amrods) [reply test/sampling.jl]: ok, i'll leave it

Comment (amrods) [reply test/sampling.jl]: thanks for your guidance

**Pull request merged**